test/extensions/filters/network/thrift_proxy/driver/README.md

```diff
@@ -0,0 +1,33 @@
+Thrift Integration Test Driver
+==============================
+
+The code in this package provides `client.py` and `server.py` which
+can be used as a thrift client and server pair. Both scripts support
+all the Thrift transport and protocol variations that Envoy's Thrift
+proxy supports (or will eventually support):
+
+Transports: framed, unframed, header
+Protocols: binary, compact, json, ttwitter/finagle
```

Comment:
typo or is it really ttwitter?

Reply:
That's what they call it: https://twitter.github.io/finagle/guide/Protocols.html?highlight=TTwitter#thrift%20user%20guide